### DISCUSSION TITLE
JavaScript code isn't displaying buttons, someone can help?

### DISCUSSION THREAD

**Discussion by schulle4u** [datenstrom/community#166] [Ask a question] [answered]
Hello Yellows, 

currently I'm working on a plugin to display the privacy-friendly Shariff social sharing buttons on my site, I've used the Fontavesome plugin as a template. 

https://github.com/schulle4u/yellow-plugin-shariff

the plugin files seem to work, all necessary css and js files are parsed and appear in the HTML output as described in the [Shariff documentation](https://github.com/heiseonline/shariff). However, the buttons don't appear on the site. Even if I disable the jQuery and Fontavesome plugins and use shariff.complete.\* files instead, no buttons are displayed, regardless where I put the `<div class="shariff"></div>` code. 

Could someone please test this and put me in the right direction again? I think only a small mistake is happening here, but I'm getting stuck. :)

Thanks
Steffen

**Comment by dieli** [comment]
Do you have an example webpage with the plugin enabled? Maybe one of the files is not found?

**Comment by schulle4u** [comment]
One of my Weblogs has the buttons enabled below the articles and
working: 

http://www.robbinaer.info/index.php?post/69

And this is an example yellow page on which I'd like the buttons to
appear: 

http://www.powerradio4u.de/blog/domain

The only difference I can see is that the first site already has Jquery
pakaged in the cms, while my yellow install currently uses a CDN version.
Jquery is enabled, I'm using it for a small icecast songstatus on the
frontpage. 

Steffen

**Comment by dieli** [comment, marked as answer]
The only difference I see is that the embedding of shariff.js looks like:
`<script src="/media/plugins/shariff.min.js" />` instead of
`<script type="text/javascript" src="/media/plugins/shariff.min.js"></script>`.
Can you modify your footer (line `if(is_file($fileNameJavascript)) $output = "<script src=\"$locationJavascript\" />\n";` in https://github.com/schulle4u/yellow-plugin-shariff/blob/master/shariff.php)?

**Comment by schulle4u** [comment]
_headdesk_ This solved the problem, many thanks! Code updated. 

Steffen
